src/ApiService/ApiService/QueueNodeHearbeat.cs

```diff
@@ -31,16 +31,25 @@ public async Task Run([QueueTrigger("myqueue-items", Connection = "AzureWebJobsS
 
         var node = await _nodes.GetByMachineId(hb.NodeId);
 
+        var log2 = log.AddTag("NodeId", hb.NodeId.ToString());
```

Comment:
Can we reuse `log` here?

```suggestion
        log = log.AddTag("NodeId", hb.NodeId.ToString());
```

Reply:
cleaning this up in next PR